net-dns/dnscap/dnscap-1.10.2.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Network capture utility designed specifically for DNS traffic"
+HOMEPAGE="https://dnscap.dns-oarc.net/"
+SRC_URI="https://www.dns-oarc.net/files/dnscap/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="ISC"
+IUSE=""
+
+RDEPEND="net-libs/libpcap
+	net-libs/ldns
+	sys-libs/zlib
```

Comment:
Next time please sort this list in alphabetical order, or in your future ebuilds. Unconditionals before conditional deps, both sorted alphabetically.